[NA][FE] fix tags column name collision in dataset items

idoberko2 opened the pull request:
## Details
Fixes the edge case where the dataset items' `data` contains a `tags` property which collides with the non-dynamic column `Tags`.

## Change checklist
- [x] User facing
- [ ] Documentation update

## Issues
- OPIK-0000

## Testing
Tested manually by reproducing the issue first, then applying the fix and making sure it fixes the issue.

## Documentation
No need, this is a bugfix

JetoPistola commented on apps/opik-frontend/src/components/pages/DatasetItemsPage/DatasetItemsPage.tsx:
```diff
@@ -215,7 +215,7 @@ const DatasetItemsPage = () => {
       .sort((c1, c2) => c1.name.localeCompare(c2.name))
       .map<DynamicColumn>((c) => ({
-        id: c.name,
+        id: `data.${c.name}`,
```
Use:

`EXPERIMENT_ITEM_DATASET_PREFIX`

It's from experiments.ts

idoberko2 commented:
@JetoPistola Done! ✅ 

Instead of reusing `EXPERIMENT_ITEM_DATASET_PREFIX`, I created a new constant `DATASET_ITEM_DATA_PREFIX` in `constants/datasets.ts` with the same value. This provides better separation and makes it clearer that this is specifically for dataset items.

**Changes in commit [`cec21c0`](https://github.com/comet-ml/opik/commit/cec21c06c):**
- Created `apps/opik-frontend/src/constants/datasets.ts` with `DATASET_ITEM_DATA_PREFIX = "data"`
- Updated `DatasetItemsPage.tsx` to import and use the new constant
- Changed from hardcoded `'data.'` to using the constant: `` `${DATASET_ITEM_DATA_PREFIX}.${c.name}` ``
